Formula/digdag.rb

```diff
@@ -0,0 +1,25 @@
+class Digdag < Formula
+  desc "Workload Automation System"
+  homepage "https://github.com/treasure-data/digdag"
+  url "https://dl.digdag.io/digdag-0.8.2.jar"
+  sha256 "9e56fe2009a8f3ef62d2e5d3d1b786d0a2a38686c846402acfdcfbb06eae2843"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "digdag-#{version}.jar" => "digdag.jar"
+    # Create a wrapper script to support OS X 10.9.
+    # See https://github.com/Homebrew/homebrew-core/pull/2012#issuecomment-226407948 for the detail.
+    (bin/"digdag").write <<-EOS.undent
+      #!/bin/bash
+      exec /bin/bash "#{libexec}/digdag.jar" "$@"
+    EOS
```

Comment:
I modified according to frsyuki's suggestion ( https://github.com/treasure-data/digdag/pull/125#issuecomment-227354246 )
If it suitable to use `write_jar_script`as discussion at the embulk formula https://github.com/Homebrew/homebrew-core/pull/2261#discussion_r67982956 , I'll fix it.